microcosm_pubsub/codecs.py

```diff
@@ -39,39 +40,17 @@ class PubSubMessageSchema(Schema):
     Base schema for messages, including a media type.
 
     """
-    media_type = fields.Method(
-        # called by dump
-        serialize="serialize_media_type",
-        # called by load
-        deserialize="deserialize_media_type",
-        attribute="mediaType",
-        # need to set missing to non-None or marshmallow won't call the deserialize function
-        missing=DEFAULT_MEDIA_TYPE,
+    MEDIA_TYPE = DEFAULT_MEDIA_TYPE
+
+    mediaType = fields.String(
```

Comment:
One piece changed here is to make encoding/decoding symmetrical and in-line with how we already do it in other libraries, like in microcosm-flask. This should make it explicit that:
* messages sent over the wire are camelCased
* loaded messages have snaked_cased attributes